services/challengedeployerservice/helper.go

```diff
@@ -117,3 +113,28 @@ func createFolder(challengeName string) (message int, challengePath string) {
 	//Successfully created directory
 	return 0, challengePath
 }
+
+func copyChallengeCheckerIntoKissaki(dirPath string, challengeName string) error {
+	srcFilePath := dirPath + "/" + challengeName + "_challenge_checker"
+	pathInPod := "/opt/kissaki/challenge-data"
+
+	log.Println("Testing" + srcFilePath + "....and..." + pathInPod)
+
+	if err := utils.CopyTarIntoPod("kissaki-0", "kissaki", pathInPod, srcFilePath, "katana"); err != nil {
+		log.Println(err)
+		return err
+	}
+	return nil
+}
+
+func copyFlagDataIntoKashira(dirPath string, challengeName string) error {
+	srcFilePath := dirPath + "/" + "flag_data"
+	pathInPod := "/opt/kashira/flag-data"
+	log.Println("Testing" + srcFilePath + "....and..." + pathInPod)
```

Comment:
either write proper logging like the scrfile is : <filepth> or remove undirected printstateents

Reply:
done